x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.tsx

```diff
@@ -35,9 +31,12 @@ export const LogEntryFieldColumn: React.FunctionComponent<LogEntryFieldColumnPro
   isHovered,
   isWrapped,
 }) => {
-  const value = useMemo(() => (isFieldColumn(columnValue) ? JSON.parse(columnValue.value) : null), [
-    columnValue,
-  ]);
+  const value = useMemo(() => {
+    if (isFieldColumn(columnValue)) {
+      return columnValue.value;
```

Comment:
Could you remind me again why we don't serialize the field values anymore?

Reply:
because REST!

Some values could be nested objects. The old GraphQL API had no way to represent this with types, so we had to serialize the values in the server and deserialize them here. With REST/io-ts we can properly model this so that step is no longer necessary.